### DISCUSSION TITLE
Curator and custom indices

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#8232] [Unsupported Versions] [answered]
Hey!
SO 2.3.130, distributed.
According to [this doc](https://docs.securityonion.net/en/2.3/curator.html#configuration) I added custom indices to `/opt/so/saltstack/local/pillar/global.sls`, restarted curator and waited cron to do it's job:
```
...
so-foo-bar:
  warm: 7
  close: 45
  delete: 365
...
```
But curator doesn't close indices older than 45 days. Moreover, in `curator.log` there is no entries with "so-foo-bar", only default ossec/zeek/etc.
What did I do wrong?

**Comment by weslambert** [comment, marked as answer]
It sounds like you need to add action files for the custom indices as well. Take a look at the ones in the following directory to get an idea of what they look like:

`/opt/so/saltstack/default/salt/curator/files/action/`

...then add them into the equivalent `local` directory.

**Reply by Sahale** [reply]
... and edited necessary files from `/opt/so/saltstack/default/salt/curator/files/bin/`
Thanks a lot!
